test/parallel/test-fs-fchown.js

```diff
@@ -52,7 +52,8 @@ function testGid(input, errObj) {
              `>= 0 && <= 2147483647. Received ${input}`
   };
   testFd(input, errObj);
-  errObj.message = 'The value of "uid" is out of range. It must be >= 0 && ' +
+  if (input === -1) return;
```

Comment:
Maybe we could just change the value from `-1` to `-2` above (and/or add that value)?